epoch-spin.diff

```diff
@@ -0,0 +1,1001 @@
+diff --git a/epoch3d/Makefile b/epoch3d/Makefile
```

Comment:
@holgerschmitz Was this accidentally included? If so, I will remove it.

Reply:
@TomGoffrey Yes, this is not intentional and can be removed.